rfcs/text/0023-match_only_text-data-type.md

```diff
@@ -28,6 +24,60 @@ The following fields are currently indexed as `text` and are candidates to migra
 * `message`
 * `error.message`
 
+ECS also has `text` type multi-fields for several fields using the convention `<field-name>.text`. This RFC also proposes to migrate all `text` multi-fields to using `match_only_text`, but the multi-field will remain using the `<field-name>.text` to avoid a breaking change.
+
+As part of this migration,
```

Comment:
I was expecting an end to this sentence after the list, is it missing?

Reply:
No, I missed it during editing. Removed.